inception-project/inception
# #764 basic results reranking for external search

**What's in the PR**
* Extract individual sentences from external search results
* Score sentences with respect to suitability for dictonary examples
* Cluster sentences based on similarity
* Display sentences in a separate view of the external search page
* resolves #764 

**How to test manually**
* Execute a query in external search with "show best sentences" view. This is done by appending " sentenceHighlights:true" to the end of the desired query.
* Observe the highlights of returned search results (should be sentences).

**Automatic testing**

**Documentation**
